lib/assessment.js

```diff
@@ -5,33 +5,57 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  var smallest = arguments[0]
+  for(var i = 1; i < arguements.length; i++){
```

Comment:
You've got a typo here: `arguements`. Typos are usually the most trivial of bugs and sometimes the most challenging to detect. You can check that several instances of the same identifier (like a variable name) are spelled the same in Sublime by highlighting one of them and looking at the rest to see that they're outlined.